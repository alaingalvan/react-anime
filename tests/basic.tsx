import React from 'react';
import renderer from 'react-test-renderer';
import Anime from '../src/react-anime';

//snapshot test
it('2 children delayed fade in', () => {
  const tree = renderer.create(
    <Anime
      easing="easeOutElastic"
      loop={true}
      duration={1000}
      direction="alternate"
      delay={(_, index) => index * 240}>
      <p>1</p>
      <p>2</p>
    </Anime>,
    {
      createNodeMock: _ => ({ nodeType: false })
    }
  );

  expect(tree.toJSON()).toMatchSnapshot();
});
