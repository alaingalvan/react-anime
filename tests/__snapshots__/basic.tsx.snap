// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 child delayed fade in 1`] = `
<DocumentFragment>
  <div>
    <p>
      1
    </p>
  </div>
</DocumentFragment>
`;
